frontend/src/app/sessions/sessions.component.html

```diff
@@ -3,14 +3,16 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-<mat-card class="m-2.5 mb-12 bg-[#010035] text-center text-white">
+<div class="collab-card m-2.5 mb-12 bg-[#010035] text-center text-white">
   <span id="welcome">Welcome to the</span>
   <h1>Capella Collaboration Manager</h1>
-</mat-card>
+</div>
 
 <app-user-sessions-wrapper>
   <div class="flex flex-wrap justify-around gap-5">
-    <app-create-persistent-session></app-create-persistent-session>
-    <app-active-sessions></app-active-sessions>
+    <app-create-persistent-session
+      class="shrink grow-[2] basis-0"
+    ></app-create-persistent-session>
+    <app-active-sessions class="shrink grow basis-0"></app-active-sessions>
   </div>
```

Comment:
Popping in at Paula's request (hi Paula 👋). I didn't check the Angular defaults to confirm it, but the Persistent Workspace Session card is 450px wide at all window widths on production so I'd just add that directly and let flex handle the spacing but not the size. I also added that width to Active Sessions because I don't know why they'd need to be different widths, though the contents of that card need ot be adjusted to use the full width (probably just a w-full on the card but I can look into it if you like).

```
<app-user-sessions-wrapper>
  <div class="flex flex-wrap justify-around">
    <app-create-persistent-session
      class="w-[450px]"
    ></app-create-persistent-session>
    <app-active-sessions class="w-[450px]"></app-active-sessions>
  </div>
</app-user-sessions-wrapper>
```
<img width="1482" alt="SessionWidths" src="https://github.com/DSD-DBS/capella-collab-manager/assets/79602065/0695e1e6-049c-4741-a4fe-e45f535f8c31">

Reply:
thanks a lot!!

Reply:
<img width="430" alt="Bildschirmfoto 2024-01-23 um 12 55 02" src="https://github.com/DSD-DBS/capella-collab-manager/assets/49535291/48494f74-c085-4389-995f-eb4242bf448d">
what would you adjust?

Reply:
Unless there's some reason for the Active Sessions element to be responsive (it only appears in the sessions homepage, right?), I think you can probably just delete .mat-card from active-sessions.component.css and let it be constrained by the w-[450px] of the entire component in the sessions.component.html so it looks the same as the persistent sessions card. But it's not essential or anything, on production they're currently different sizes, it's really only noticable between like 500-900px screen width, but putting their widths in the same spot might make it more consistent or easier ot update later.

Reply:
alright I see - thank you!

Reply:
I think that can be an enhancement for the future as the button above `Open internal session viewer` does not scale the same as the underlying card and thus would result in funny looking results